src/installation/index.md

```diff
@@ -1,160 +1,204 @@
+> 2. Rust compiler forks. If `LLVM` changes are accepted, we will proceed with the Rust compiler changes.
+
+
+[llvm-github-fork]: https://github.com/espressif/llvm-project
+[gcc-toolchain-github-fork]: https://github.com/espressif/crosstool-NG/
+[rustup-overrides]: https://rust-lang.github.io/rustup/overrides.html
+[llvm-github-fork-upstream issue]: https://github.com/espressif/llvm-project/issues/4
+
+
+## `std` Development Requirements
+
+Regardless of the target architecture, make sure you have the following required tools installed to build [`std`][rust-esp-book-overview-std] applications:
+
+- [`python`][python-website-download]: Required by ESP-IDF
+- [`git`][git-website-download]: Required by ESP-IDF
+- [`ldproxy`][embuild-github-ldproxy] crate: Simple tool to pass the `ldproxy` linker arguments to the actual linker executable. Install it  by running
   - `cargo install ldproxy`
 
+The std runtime uses [ESP-IDF][esp-idf-github] (Espressif IoT Development Framework) as hosted environment but, users do not need to install it. ESP-IDF is automatically downloaded and installed by [esp-idf-sys][esp-idf-sys-github], a crate that all std projects need to use, when building a std application.
+
+
+[rust-esp-book-overview-std]: ../overview/using-the-standard-library.md
+[python-website-download]: https://www.python.org/downloads/
+[git-website-download]: https://git-scm.com/downloads
+[embuild-github-ldproxy]: https://github.com/esp-rs/embuild/tree/master/ldproxy
+[esp-idf-sys-github]: https://github.com/esp-rs/esp-idf-sys
+[esp-idf-github]: https://github.com/espressif/esp-idf
 
-[ESP-IDF]: https://github.com/espressif/esp-idf
-[`std` approach]: ../overview/using-the-standard-library.md
-[`ldproxy`]: https://github.com/esp-rs/embuild/tree/master/ldproxy
-[esp-rs/embuild]: https://github.com/esp-rs/embuild
-[ESP-IDF installation note]: https://github.com/esp-rs/espup#esp-idf-instalation
 
 ## Using Containers
 
-As an alternative to installing the environment directly on your local system, it's also possible to run it inside a container.
+Instead of installing directly on your local system, you can host the development environment inside a container. Espressif provides the [idf-rust] image that supports both `RISC-V` and `Xtensa` target architectures and enables both `std` and `no_std` development.
+
+You can find numerous tags for `linux/arm64` and `linux/amd64` platforms.
+
+For each Rust release, we generate the tag with the following naming convention:
+
+- `std` applications
+  - `<chip>_<esp-idf-version>_<rust-toolchain-version>`
+  - For example, `esp32s3_v4.4_1.64.0.0` contains the ecosystem for developing `std` applications based on [ESP-IDF release/v4.4] for `ESP32-S3` with the `1.64.0.0` Rust toolchain.
+- `no_std` applications
+  - `<chip>_<rust-toolchain-version>`
+  - For example, `esp32_1.64.0.0` contains the ecosystem for developing `no_std` applications for `ESP32` with the `1.64.0.0` Rust toolchain
```

Comment:
```suggestion
- `<chip>_<rust-toolchain-version>`
  - For example, `esp32_1.64.0.0` contains the ecosystem for developing `std`, and `no_std` applications for `ESP32` with the `1.64.0.0` Xtensa Rust toolchain.
```

Reply:
The same thing with `, and` between noun phrases: the comma is not required in such cases in English. For details, see [this](https://www.scribbr.com/commas/comma-before-or-after-and/).